Net Runner/ModelOutput/ModelOutputManager.m

```diff
@@ -52,6 +52,8 @@ - (instancetype)initWithClasses:(NSDictionary*)classes {
     NSMutableDictionary<NSString*,Class> *classes = [[NSMutableDictionary alloc] init];
     
     classes[@"image.classification.imagenet"] = NSClassFromString(@"ImageNetClassificationModelOutput");
+    classes[@"image.classification.emotion"] = NSClassFromString(@"ClassificationModelOutput");
```

Comment:
@phildow : Why does this have to be done in code (as opposed to handled based on configuration in the `model.json` file)?

Reply:
actual formatting has to be done in code, but if the formatter is generic enough it could make sense to specify the formatter to use in the json’s options field. i’ll take a look.